### ISSUE TITLE
Errors on removing failed job

### ISSUE THREAD

**Issue by wilbertverayin** [anyTV/anytv-node-kue#2]
when using the analyzer these error come up from time to time
```
2017-05-24 02:09:25:191ms - error: Error in removing failed job 74478 Error: job "74478" doesnt exist
    at Command.callback (/home/wilbert/projects/channel-analyzer/node_modules/kue/lib/queue/job.js:175:17)
    at normal_reply (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:721:21)
    at RedisClient.return_reply (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:819:9)
    at JavascriptRedisParser.returnReply (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:192:18)
    at JavascriptRedisParser.execute (/home/wilbert/projects/channel-analyzer/node_modules/redis-parser/lib/parser.js:574:12)
    at Socket.<anonymous> (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:274:27)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:560:20)
```

```
/home/wilbert/projects/channel-analyzer/node_modules/anytv-kue/lib/AnyTVKue.js:139
                            job.inactive();
                               ^

TypeError: Cannot read property 'inactive' of undefined
    at /home/wilbert/projects/channel-analyzer/node_modules/anytv-kue/lib/AnyTVKue.js:139:32
    at Command.callback (/home/wilbert/projects/channel-analyzer/node_modules/kue/lib/queue/job.js:175:14)
    at normal_reply (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:721:21)
    at RedisClient.return_reply (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:819:9)
    at JavascriptRedisParser.returnReply (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:192:18)
    at JavascriptRedisParser.execute (/home/wilbert/projects/channel-analyzer/node_modules/redis-parser/lib/parser.js:574:12)
    at Socket.<anonymous> (/home/wilbert/projects/channel-analyzer/node_modules/redis/index.js:274:27)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:560:20)
```

We can suppress the second error by having 
`job && job.inactive();` on Line 139 on `lib/AnyTVKue.js`

Dunno how to deal with the first one though.

**Comment by pandafox** [comment]
Do we already set to delete jobs on completion? Perhaps that's' what triggers the error? Or do we have problem with re-try/re-queuing?

**Comment by FerrielMelarpis** [comment]
@pandafox jobs are deleted on completion unless set on the base config of the library.
Already filed a bug ticket https://freedom.myjetbrains.com/youtrack/issue/Delta-62
This line causes that behavior https://github.com/anyTV/anytv-node-kue/blob/7acaf09e4af067d711b8798e88e5853ce8f45262/src/AnyTVKue.js#L41

**Comment by pandafox** [comment]
I think those are two separate problems, as anyone process can remove the job, so we would still need to check if the job exists as @wilbertverayin suggested. Perhaps fix both of them in the same PR?
https://github.com/anyTV/anytv-node-kue/blob/master/src/AnyTVKue.js#L106

**Comment by FerrielMelarpis** [comment]
This should have been fixed by #3. https://github.com/anyTV/anytv-node-kue/pull/3/files#diff-ab0471b1be4e8acd9be94c7a67dbbaf1R60
